db/migrate/20220914075650_create_likes.rb

```diff
@@ -1,8 +1,8 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
-      t.references :user, null: false, foreign_key: { to_table: :users }
-      t.references :post_id, null: false, foreign_key: { to_table: :posts }
```

Comment:
[OPTIONAL] It looks like you didn't add MIT license to your project. When I tried to open the link from the Readme file it shows me a broken link. Kindly fix this before merging :).